scripts/release.sh

```diff
@@ -142,6 +142,12 @@ main() {
         exit 1
       fi
       git tag --local-user "${KEYID}" --sign "${RELEASE_VERSION}" --message "${RELEASE_VERSION}"
+
+      if [ "$DRY_RUN" != "true" ]; then
+        read -p "Push tag ${RELEASE_VERSION} to github.com/etcd-io/etcd [y/N]? " -r confirm
```

Comment:
Should this be using `${REPOSITORY}` variable ?

```suggestion
        read -p "Push tag ${RELEASE_VERSION} to ${REPOSITORY} [y/N]? " -r confirm
```

Reply:
Agreed. I'll update the previous line with the same output.